indico/modules/events/papers/templates/display/_common.html

```diff
@@ -1,14 +1,18 @@
-{% macro render_cfp_infoline(event, cfp, contributions, show_icon=true) %}
```

Comment:
AFAICT this was copied from CfA where we show the icon where we render the infoline also on the main conference page, which we don't do with CfP. The only place where it's rendered sets `show_icon` to False so I just removed it